composer.json

```diff
@@ -2,12 +2,12 @@
     "name": "ubccr/ralph",
     "description": "A simple SPARQL client built on top of Guzzle.",
     "type": "library",
-    "keywords": ["sparql","rdf"],
+    "keywords": ["rdf", "sparql"],
     "homepage": "https://github.com/ubccr/ralph",
     "license": "LGPL-3.0-only",
     "authors": [
         {
-            "name": "Center for Computational Research, the University at Buffalo",
+            "name": "Center for Computational Research, SUNY the University at Buffalo",
```

Comment:
The official name is "Center for Computational Research, University at Buffalo. The State University of New York"

Reply:
I get the distinct feeling there was a serious meeting where a committee debated the formatting of that name.

"Should that be a comma here, or a period?"